assets/eip-4824/daostar_context.jsonld

```diff
@@ -0,0 +1,7 @@
+{
+    "@context":
+    {
+        "@vocab": "http://daostar.org/schemas/",
+        "type": "@type"
+    }
+ }
```

Comment:
```suggestion
{
    "@context": {
        "@vocab": "http://daostar.org/schemas/",
        "type": "@type"
    }
}

```
I know it's not a rule in EIP-1, but... can we agree this is how JSON *should* be formatted?

Reply:
While I personally use curly on same line and I support you suggesting it, I don't think we should dictate coding style in EIPs and thus I would consider this a non-blocker.